jsk_pcl_ros_utils/scripts/xyz_to_screenpoint.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import rospy
+
+from sensor_msgs.msg import CameraInfo
+from geometry_msgs.msg import PointStamped
+
+from image_geometry import PinholeCameraModel
+
+class XYZToScreenPoint:
```

Comment:
please inherit object

```suggestion
class XYZToScreenPoint(object):
```